python-runnables/add-node-pool/runnable.py

```diff
@@ -59,6 +59,8 @@ def run(self, progress_callback):
         node_pool = self.config.get('nodePool', {})
         if 'machineType' in node_pool:
             args = args + ['--node-type', node_pool['machineType']]
+         if 'securityGroup' in node_pool:
```

Comment:
also check that `node_pool['securityGroup']` is not a empty string (not `None` but `len(...) == 0`)